.ci/pytorch/cpp_doc_push_script.sh

```diff
@@ -58,6 +58,7 @@ time python tools/setup_helpers/generate_code.py \
 
 # Build the docs
 pushd docs/cpp
+pip install -e git+https://github.com/pytorch/pytorch_sphinx_theme.git@pytorch_sphinx_theme2#egg=pytorch_sphinx_theme2
```

Comment:
It would be nice to move it to some sort of common.sh and call function from there, rather than repeating the same string thrice